### ISSUE TITLE
[Bug]: transporter.verify(...).then is not a function was not fixed in 2.0.0

### ISSUE THREAD

**Issue by alwyn974** [nest-modules/mailer#1172]
**Describe the bug**
```js
transporter.verify(...).then is not a function TypeError: transporter.verify(...).then is not a function
```

As stated in the release changelog this bug was fixed in #1153
But in npm the code doesn't match the release https://www.npmjs.com/package/@nestjs-modules/mailer?activeTab=code
The fix is present in the tagged release https://github.com/nest-modules/mailer/blob/621621a9ce10e78b5083453ad98bde16636305b2/lib/mailer.service.ts#L132

```js
verifyTransporter(transporter, name) {
        const transporterName = name ? ` '${name}'` : '';
        transporter.verify()
            .then(() => this.mailerLogger.error(`Transporter${transporterName} is ready`))
            .catch((error) => this.mailerLogger.error(`Error occurred while verifying the transporter${transporterName}}: ${error.message}`));
    }
```

**Comment by juandav** [comment]
I'm working on some improvements and fixes that I've found, and I'm also transitioning the project to use TurboRepo so it can be tested more effectively.

**Comment by juandav** [comment]
https://github.com/nest-modules/mailer/pull/1173

**Comment by alwyn974** [comment]
Fixed with 2.0.1 release https://www.npmjs.com/package/@nestjs-modules/mailer/v/2.0.1
